src/Layout/ComputationMenu/ConnectedComputationMenu.tsx

```diff
@@ -133,6 +147,8 @@ const mapDispatchToProps = (
     saveComparison: () => {
         const saveComparisonAction = saveComparison();
         dispatch(saveComparisonAction);
+        const saveSettingsAction = saveSettings();
```

Comment:
Why is it necessary to save/restore settings?

Reply:
Because we want to track changes when we save/restore specific computations, not only changes against the historical settings.